sql/sql_audit.cc

```diff
@@ -360,6 +362,8 @@ class Ignore_event_error_handler : public Audit_error_handler {
   const char *m_event_name;
 };
 
+std::string cmd_class_lowercase;
+
```

Comment:
Global string object with concurrent threads modifying and accessing it is still not acceptable.

Reply:
Thanks for getting rid of this.